tasks/page/run_ibom

```diff
@@ -31,10 +32,11 @@ board = None
 ext = os.path.splitext(board_file)[1]
 if ext == '.brd':
     tmp_board = pcbnew.LoadBoard(board_file, pcbnew.IO_MGR.EAGLE)
-    with tempfile.TemporaryDirectory() as tmpdir:
-        tmp_board_file = os.path.join(tmpdir, 'eagle_import.kicad_pcb')
-        pcbnew.SaveBoard(tmp_board_file, tmp_board)
-        board = pcbnew.LoadBoard(tmp_board_file)
+    tmpdir = tempfile.mkdtemp()
+    tmpfile = os.path.join(tmpdir, 'import_{}.kicad_pcb'.format(os.getpid()))
+    pcbnew.SaveBoard(tmpfile, tmp_board)
+    board = pcbnew.LoadBoard(tmpfile)
+    shutil.rmtree(tmpdir, ignore_errors=True)
```

Comment:
Could you explain what's going on here in the commit message please?

Reply:
I'll put a comment in the code instead, to make it clear what the reason is for saving and reloading the board.